test/selenium/selenium-kdump.py

```diff
@@ -83,6 +84,7 @@ def testNFS(self):
                         "Remote over NFS",
                         {"kdump-settings-nfs-mount": "localhost:{}".format(self.share)},
                         )
+        self.refresh(self.frame_name)
```

Comment:
Indeed I often noticed that the page is out of date. It just looks like this reload is covering up a bug? I. e. the kdump page doesn't update itself properly? Or is there some specific reason why this wouldn't work under selenium?

Reply:
@martinpitt I have no idea, I have a fear that there could be some bug maybe underneath connected with js operations and changing ``kdump`` via cockpit UI.

I've added this page reload to enforce that page is up to date, regarding to current state of kdump service, and not contains some messy info.
When I've tested it locally, changing to NFS kdumps sometimes staled  and timeouts, there is trouble that it regenerates initrd, and in case somegthing fails, could cause kdump issue. but t is not raised as an error.